test/numbers.jl

```diff
@@ -830,6 +830,54 @@ end
     @test cmp(isless, 1, NaN) == -1
     @test cmp(isless, NaN, NaN) == 0
 end
+@testset "ispositive/isnegative" begin
+    for T in Base.uniontypes(Base.IEEEFloat)
```

Comment:
```suggestion
@testset "ispositive/isnegative" begin
    @testset "Type $(T)" for T in Base.uniontypes(Base.IEEEFloat)
```
and same below, so that people can more easly tell what type is failing

Reply:
reopened since other tests need this